README.md

```diff
@@ -57,28 +57,33 @@ Metrics that don't match any mapping in the configuration file are translated
 into Prometheus metrics without any labels and with certain characters escaped
 (`_` -> `__`; `-` -> `__`; `.` -> `_`).
 
-In general, the different metric types are translated as follows, with certain
-suffixes appended to the Prometheus metric names:
+In general, the different metric types are translated as follows:
 
-    StatsD gauge   -> Prometheus gauge (suffix `_gauge`)
+    StatsD gauge   -> Prometheus gauge
 
-    StatsD counter -> Prometheus counter (suffix `_counter`)
+    StatsD counter -> Prometheus counter
 
-    StatsD timer   -> Prometheus summary (suffix `_timer`)        <-- indicates timer quantiles
-                   -> Prometheus counter (suffix `_timer_total`)  <-- indicates total time spent
-                   -> Prometheus counter (suffix `_timer_count`)  <-- indicates total number of timer events
+    StatsD timer   -> Prometheus summary                    <-- indicates timer quantiles
+                   -> Prometheus counter (suffix `_total`)  <-- indicates total time spent
+                   -> Prometheus counter (suffix `_count`)  <-- indicates total number of timer events
 
-An example mapping configuration:
+If `-statsd.add-suffix` is set, the exporter appends the metric type (gauge,
```

Comment:
Be more explicit/precise about what they can expect: "(`_gauge`, `_counter`, `_timer`)".